src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return Colors.values()[new Random().nextInt(Colors.values().length)].toString();
```

Comment:
It is better to put random into a class variable, as in the example: 

![image](https://user-images.githubusercontent.com/12864128/217531676-7c170868-206e-417b-89c7-2c037a3daf1d.png)